Plugins/BTCPayServer.Plugins.Blink/BlinkLightningClient.cs

```diff
@@ -298,32 +299,48 @@ public async Task<LightningPayment[]> ListPayments(CancellationToken cancellatio
     public async Task<LightningInvoice> CreateInvoice(CreateInvoiceParams createInvoiceRequest,
         CancellationToken cancellation = new())
     {
-        var reques = new GraphQLRequest
-        {
-            Query = @"
-mutation LnInvoiceCreate($input: LnInvoiceCreateInput!) {
-  lnInvoiceCreate(input: $input) {
+        string btcLnInvoiceCreate = @"
+mutation LnInvoiceCreate($input: LnInvoiceCreateOnBehalfOfRecipientInput!) {
+  lnInvoiceCreateOnBehalfOfRecipient(input: $input) {
```

Comment:
Switched this one to be the non-authed `lnInvoiceCreateOnBehalfOfRecipient` mutation